crates/binjs_io/src/entropy/rw.rs

```diff
@@ -161,9 +163,12 @@ pub type NameData = SmallVec<[u8; NAME_MAX_LEN]>;
 ///
 /// `TableRef` values are represented as follows:
 /// - a value `0` represents an `TableRef::Prelude` with index 1 + the latest `TableRef::Prelude` value we have encoded/decoded;
-/// - a value `v` in `[1, window len]` represents a repeat of a previous value: 1 it the latest value, 2 the second latest, etc.
-/// - a value `v` in `[window len + 1, window len + length of the shared dictionary]` represent `TableRef::Shared` with index `v - window len - 1`;
-/// - a value `v > window len + length of the shared dictionary` represents `TableRef::Prelude` with index `v - window len - length of shared dictionary - 1`.
+/// - a value `v` in `[1, window len]` represents a repeat of a previous value: 1 for the latest value, 2 the second latest, etc.
+/// - a value `v` in `[window len + 1, window len + prelude len]` represents a `TableRef::Prelude` with index `v - window len - 1`;
+/// - a value `v` in `[window len + prelude len + 1, window len + prelude len + number of distinct TableRef::Shared encountered so far]`
+///       represents the `v - prelude len - window len`th `TableRef::Shared` encountered by this `TableRefStreamState`;
+/// - a value `v > window len + prelude len + number of distinct TableRef::Shared encountered so far` represents a fresh `TableRef::Shared`
+///       with index `v - window len + prelude len + number of distinct TableRef::Shared encountered so far`.
```

Comment:
can these comments be wrapped into 80 chars or so?
(that would make review easier, given I don't have to scroll horizontally for each line :P )